include/RAJA/policy/openmp/atomic.hpp

```diff
@@ -88,7 +88,7 @@ RAJA_SUPPRESS_HD_WARN
 template<typename T>
 RAJA_HOST_DEVICE RAJA_INLINE T atomicMin(omp_atomic, T* acc, T value)
 {
-#if _OPENMP >= 202011
+#if _OPENMP >= 202011 && defined(RAJA_OPENMP_5_1_ATOMICS)
```

Comment:
Could you just use `RAJA_ENABLE_OPENMP_5_1_ATOMICS` directly here instead of adding a secondary macro `RAJA_OPENMP_5_1_ATOMICS`?

Reply:
Yes, I thought the same thing. But I see examples where we do this in a few places ,for example: 

```
#if defined(RAJA_ENABLE_BOUNDS_CHECK)
#define RAJA_BOUNDS_CHECK_INTERNAL
#define RAJA_BOUNDS_CHECK_constexpr
#else
#define RAJA_BOUNDS_CHECK_constexpr constexpr
#endif
```

```
#if defined(RAJA_ENABLE_OPENMP_TASK)
#define RAJA_ENABLE_OPENMP_TASK_INTERNAL
#endif
```

I'm not sure why we do this, but if we want to keep it consistent maybe I add to  the secondary macro _INTERNAL. 

What do others think?

Reply:
Maybe add the check for _OPENMP being defined and >= 202011 when defining the internal macro (maybe named RAJA_USE_OPENMP_5_1_ATOMICS?), and then the actual atomic implementation could simply use the internal macro instead of checking the openmp version.

Reply:
Or maybe the configuration variable should be RAJA_ALLOW_OPENMP_5_1_ATOMICS and the internal define should be RAJA_ENABLE_OPENMP_5_1_ATOMICS. Just brainstorming.

Reply:
Sounds good to me!